backend/routes/verifiedusers.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const Router = require('koa-router');
+const parser = require('koa-body');
+
+const Users = require('../models/users');
+
+const parsers = {
+    query: parser({urlencoded: true, multipart: false, json: false}),
+    form: parser({urlencoded: false, multipart: true, json: false}),
+    json: parser({urlencoded: false, multipart: false, json: true}),
+};
+
+const verifiedUsers = new Router();
+
+verifiedUsers.get('/', async ctx => {
```

Comment:
I don't feel strongly about this query having its own endpoint.. just curious, was there an explicit motivation to not put it in `users.js`? We might come back to this when we have more queries but this might be the path to go down, I don't know yet.

Reply:
I was just having errors having 2 user GET endpoints - with the request hitting the `users/:id` endpoint rather than `users/verifiedusers` or whatever. Sure there is a way to do that, I'm just newer to making routes.